tests/migration/postcopy.go

```diff
@@ -208,6 +208,12 @@ var _ = SIGMigrationDescribe("VM Post Copy Live Migration", func() {
 					return fmt.Errorf("waiting for virt-handler pod to come back online")
 				}, 2*time.Minute, 1*time.Second).Should(Succeed(), "Virt handler should come online")
 			}
+
+			AfterEach(func() {
+				By("Ensuring the virt-handler killer pod is removed")
+				removeVirtHandlerKillerPod()
+			})
```

Comment:
I think this is already taken care of by:
https://github.com/kubevirt/kubevirt/blob/9c9782a62bd63817b8103b0fa863b45c626b974f/tests/testsuite/namespace.go#L153-L154